unit/doc.go

```diff
@@ -72,18 +72,6 @@
 // used, however, to create the unit of 'Slide', because in this case slide
 // is just a measurement of area. Instead, a constant could be defined.
 //		const Slide unit.Area =  0.001875 // m^2
-//		var WhiteBloodCellDim unit.Dimension
```

Comment:
The plan is to make this a real example eventually, but this will come after the fix to the `Mul` method.